social-network/src/YourArticle/YourArticle.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```

Comment:
YourArticle and MissedArticle could've been one component with a minor variation in the layout.